animations/src/test/java/com/chiralbehaviors/CoRE/workspace/dsl/TestImport.java

```diff
@@ -44,4 +46,27 @@ public void testExampleWorkspace() throws Exception {
         assertNotNull(workspace.getScope()
                                .lookup("kernel", "IsA"));
     }
+
+    @Test
+    public void testIncrementalVersionUpdate() throws Exception {
```

Comment:
Test that we can successfully incrementally load a new version of the workspace.